test/unit/helpers/wsApi.js

```diff
@@ -13,188 +13,111 @@ var wsApi = require('../../../helpers/wsApi');
 var failureCodes = require('../../../api/ws/rpc/failureCodes');
 
```

Comment:
I would like to group in context each parameter.

Reply:
it would be repeated with text description then: ` 
version
    should return an error when version is float
`

Reply:
I'm saying something like:
```
version
      should return error when using float

Reply:
done